package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "language-negotiator-lambda",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
I bumped to full v1.0 because this is our first version.  The previous 0.X was to mark that it was not feature complete yet